docs/faq_setup.md

```diff
@@ -48,3 +48,12 @@ You can [request a demo here](https://www.vortexa.com/request-demo-sdk).
 More details are given in [docs.vortexa.com](https://docs.vortexa.com/reference/intro-authentication).
 
 Alternatively, please get in touch at [www.vortexa.com](https://www.vortexa.com/).
+
+
+#### How can I check the VortexaSDK is setup correctly?
+
+You can run the `check_setup.py` file from your IDE, or from a bash console:
+
+```
+$ python -m vortexasdk check-setup 
```

Comment:
Same suggestion:
```suggestion
$ python -m vortexasdk.check-setup 
```

Reply:
That way means we can remove the __main__ function, nice.

Reply:
Interesting, the original way used vortexasdk with argparser etc. We can remove this entirely, and just call the module directly. 👍